SKYKitChat/Classes/Core/SKYChatExtension.m

```diff
@@ -553,7 +553,7 @@ - (void)fetchMessagesWithConversationID:(NSString *)conversationId
                      arguments:arguments
              completionHandler:^(NSDictionary *response, NSError *error) {
                  if (error) {
-                     NSLog(@"error calling hello:someone: %@", error);
+                     NSLog(@"error calling chat:get_messages: %@", error);
```

Comment:
While this PR is okay, I don’t think the completion handler (`completion`) is called correctly.

Suppose in the unlikely scenario when `chat:get_messages` return zero messages, `completion(nil, error)` is called with both parameter being nil. In this scenario the first parameter should be empty `NSArray`.

Let me know if you think this is out of scope and I will merge this.